Makefile

```diff
@@ -35,13 +37,20 @@ uninstall: ## Uninstall Spyglass
 	@printf "\nDisable spyglass service ... \n"
 	@sudo systemctl disable spyglass
 	@printf "\nRemove systemd service file ...\n"
-	@sudo rm -f $(SYSTEMD)/spyglass.service
+	@sudo unlink -f $(SYSTEMD)/spyglass.service
```

Comment:
Why do you want to use `unlink` instead of `rm`?
Also `unlink` doesn't take any parameters

Reply:
lol, to be fair I dont really know why I changed it to unlink 🥲